### PR TITLE
[CLHC-101111] Sync all the config changes made in cube-workers into multi group

### PR DESCRIPTION
**Pull request by tgadde-CHT**

### CODE REVIEW

**Review by mclement-vmw — APPROVED** [review]

**Review by sbhavinCH — DISMISSED** [review]

**Review by emjaymikes — DISMISSED** [review]

**Review by emjaymikes — APPROVED** [review]

**Comment by tgadde-CHT** [comment]
adding @kurajiv  for PE visiblity

File: service/charts/pgb-cube-workers/values.use1.preprod/settings.yml [hunk 1]
```diff
@@ -37,13 +37,13 @@ s3:
   chealth-s3-access-id: {{ .Values.aws_keys.chealth_s3_access_id }}
   chealth-s3-secret-key: {{ .Values.aws_keys.chealth_s3_secret_key }}
   chealth-s3-bucket: cloudhealth-detailed-bills-dev
-  billing_bucket: cloudhealth-detailed-bills-aws-use1-preprod-verify
+  billing_bucket: cloudhealth-detailed-bills-use2-preprod-test
```
Comment (mclement-vmw) [inline service/charts/pgb-cube-workers/values.use1.preprod/settings.yml:40]: I'm probably just confused, but is it okay for the bucket to have `use2` when these are settings for `use1`? I'm not sure if the cluster and bucket need to match up, but I wanted to make sure this was okay

Comment (tgadde-CHT) [reply service/charts/pgb-cube-workers/values.use1.preprod/settings.yml:40]: yea these changes are specifically for AWS pre-prod for places where we sync back from GCP 
Screenshot 2024-09-12 at 1 38 28 PM

**Review by mclement-vmw — APPROVED** [review]

**Comment by tgadde-CHT** [comment]
Closing this PR, running into RMQ auth error, will try to open another PR hoping it will resolve may be this node `gke-svchtg1-saas-gke5-vmwcht-green-4feda167-h9nj` is busted?`

```
 Bunny::AuthenticationFailureError: Authentication with RabbitMQ failed. Please check your connection settings. Username: default_user_vhost: /, password length: 32
.1.48756089808320.[2024-09-13 00:39:59] ERROR CloudHealth: /home/cloudhealth/bundle/gems/bunny-2.0.1/lib/bunny/session.rb:997:in `open_connection'
/home/cloudhealth/bundle/gems/bunny-2.0.1/lib/bunny/session.rb:280:in `start'
/home/cloudhealth/bundle/bundler/gems/cht_shared-013f2975f6ed/lib/cht_shared/queues/amqp_connection.rb:90:in `init_connection'
/home/cloudhealth/bundle/bundler/gems/cht_shared-013f2975f6ed/lib/cht_shared/queues/amqp_connection.rb:16:in `initialize'
/home/cloudhealth/bundle/bundler/gems/cht_shared-013f2975f6ed/lib/cht_shared/cubes/cube_amqp_connection.rb:53:in `initialize'
/home/cloudhealth/bundle/bundler/gems/cht_shared-013f2975f6ed/lib/cht_shared/cubes/cube_amqp_connection.rb:48:in `new'
/home/cloudhealth/bundle/bundler/gems/cht_shared-013f2975f6ed/lib/cht_shared/cubes/cube_amqp_connection.rb:48:in `new_connection'
/root/cp-workers/lib/cubes/manager_partner_super_preprocessing.rb:76:in `new_connection'
/home/cloudhealth/bundle/bundler/gems/cht_shared-013f2975f6ed/lib/cht_shared/queues/manager.rb:12:in `initialize'
/root/cp-workers/lib/cubes/manager_partner_super_preprocessing.rb:5:in `initialize'
/root/cp-workers/lib/cubes/manager_partner_super_aws_preprocess
```

**Pull request closed without merging**